KsApi/extensions/NSURLSession.swift

```diff
@@ -35,7 +35,8 @@ internal extension URLSession {
         guard let response = response as? HTTPURLResponse else { fatalError() }
 
         /// `error` is `nil` or `handleError` returns `false`.
```

Comment:
Another outdated comment referring to `handleError` - please update. Also, in this comment, can you expand on why we want to process the response when the SDK couldn't handle the response? I would've expected that to be a failure state and the `true` to be a continue state, but I'll admit I don't think the documentation is very clear here. It's also very possible that `true` indicates that the error is fully handled and we don't need to do anything else. Is this something you've been able to test (where the error is non-nil)?

Reply:
_What?!_

Reply:
Ok I'm going to delete the inline comment because I don' think we need it. 
In fact - we generally don't write comments in line unless its' `/// TODO:` or `/// FIXME:`
This was just something left in from the previous implementation so I think I'll just remove it.